src/CSSList/AtRuleBlockList.php

```diff
@@ -57,9 +57,11 @@ public function __toString()
     }
 
     /**
+     * @param OutputFormat $oOutputFormat
+     *
      * @return string
      */
-    public function render(OutputFormat $oOutputFormat)
+    public function render($oOutputFormat)
```

Comment:
This doesn't look right.  It seems to be reverting a previous change.

Reply:
Yes, it's removing the type declaration.

The reason for this that there are two possible ways to fix the implicit nullable parameters:

1. make the parameters explicitly nullable using a native type declaration (which is not possible with PHP 5.6, which we support in this branch)
2. drop the native type declaration for this parameter and use PHPDoc type annotations instead

I'll create a separate pre-PR to do 2. first (without changing the type).

Reply:
There is another way, which is to make a 9.0 release with where we're at.  The only breaking change I'm aware of is requiring PHP >= 7.2.